github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```
@@ -1650,25 +1678,15 @@ presubmits:
     annotations:
       fork-per-release: "true"
       testgrid-dashboards: kubevirt-presubmits
-    cluster: prow-workloads
+    cluster: prow-arm64-workloads
     decorate: true
     decoration_config:
       grace_period: 5m0s
       timeout: 4h0m0s
-    extra_refs:
-    - base_ref: main
-      org: kubevirt
-      repo: project-infra
     labels:
-      preset-bazel-cache: "true"
-      preset-bazel-unnested: "true"
-      preset-docker-mirror-proxy: "true"
-      preset-github-credentials: "true"
-      preset-kubevirtci-quay-credential: "true"
-      preset-pgp-bot-key: "true"
+      preset-bazel-unnested: "false"
       preset-podman-in-container-enabled: "true"
-      preset-shared-images: "true"
-    max_concurrency: 1
+    max_concurrency: 5
     name: pull-kubevirt-e2e-arm64
     optional: true
     skip_branches:
```

Let's first not report to GitHub to eventually avoid confusion for users.

```suggestion
    skip_report: false
    skip_branches:
```